posydon/interpolation/IF_interpolation.py

```diff
@@ -397,6 +397,10 @@ def __init__(self, grid=None, in_keys=None, out_keys=None, in_scaling=None,
             A list of classes that the simulation tracks can
             fall into. Usually specified as the mass transfer type. This only
             needs be specified if interp_method is a list.
+        valid_classes : list of strings
```

Comment:
Why do you add a docstring for a Parameter, which is not supported by/add to this function?